MAUI/ImageEditor/crop.md

```diff
@@ -29,29 +29,83 @@ The [Crop](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.SfIma
  The following code shows cropping the image to any desired size.
 
 {% tabs %}
-{% highlight C# tabtitle="C#" %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+
+   <Grid>

```

Comment:
Use row definition in a single row, update this for all codes

 <Grid RowDefinitions="0.9*, 0.1*">

Reply:
Updated all the codes as you mentioned above